articles/operator-nexus/quickstarts-virtual-machine-deployment-cli.md

```diff
@@ -45,6 +45,9 @@ Before you run the commands, you need to set several variables to define the con
 | ACR_USERNAME               | The username for the Azure Container Registry.                                                        |
 | ACR_PASSWORD               | The password for the Azure Container Registry.                                                        |
 
+> [!IMPORTANT]
+> Please ensure that any sensitive data (e.g. secrets, passwords, private keys, etc.) are encrypted before they are submitted in the userdata or networkdata fields.
```

Comment:
@matthew-fuller Can you provide an example that demonstrates how to encrypt and decrypt data? Specifically, how does a user encrypt data, which key should be used (e.g., public or private key), and how is it decrypted? As a user, I'm unclear about the steps involved in encrypting and decrypting data for application use. Could you clarify this process?